oaipmhTemplates/ariadne/ariadneContributor.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+    <aocat:has_contributor count="+">
```

Comment:
It won't work like that. The `count` makes sense only for node with `val` (like you already do in the node importing this subtemplate in https___vocabs_acdh_oeaw_ac_at_schema_TopCollection.xml).
Applies to all other subtemplates.
Anyway it shouldn't cause any error.